docs/pages/1 - Intro to Mill.md

```diff
@@ -15,19 +15,35 @@ programming language, and can serve as a replacement for
 other language or platform via modules (written in Java or Scala) or through
 external subprocesses.
 
+## Installation
+
+### Arch Linux
+
+Arch Linux has an [AUR package for mill](https://aur.archlinux.org/packages/mill/):
+
+```bash
+pacaur -S mill
```

Comment:
or `yaourt -S  mill` I guess ?

Reply:
is `pacaur ` is some newer alternative to yaourt?

Reply:
Yes, `pacaur` is newer and IMO the better alternative.